README.md

```diff
@@ -1,4 +1,13 @@
 # @apostrophecms/event
 Note: This module is in Alpha and not ready for use. It is a community-supported bundle and contributions are welcome!
 
-This bundle is intended to provide event functionality (concerts, recurring webinars, etc.) with <a href="https://apostrophecms.com/">Apostrophe CMS 3</a>. It is based on the `apostrophe-events` bundle for Apostrophe CMS 2. 
+This bundle is intended to provide event functionality (concerts, recurring webinars, etc.) with <a href="https://apostrophecms.com/">Apostrophe CMS 3</a>. It is based on the `apostrophe-events` bundle for Apostrophe CMS 2.
+
+## Stable release punch list
+- [ ] The index page should only show upcoming published events
```

Comment:
Unless filtered by year or month, at which point it shows all events whose span of time touches that year or month.

Reply:
In the A2 code that was being ported, the upcoming filter (now query builder) defaults to true, and is cancelled automatically if you set the year or month filters.